src/file/mod.rs

```diff
@@ -189,7 +189,10 @@ impl TempPath {
     /// If this method fails, it will return `self` in the resulting
     /// [`PathPersistError`].
     ///
-    /// Note: Temporary files cannot be persisted across filesystems.
+    /// Note: Temporary files cannot be persisted across filesystems. Also
+    /// neither the file contents nor the containing directory are
+    /// synchronized, so the update may not yet have reached the disk when
+    /// `persist` returns.
```

Comment:
Can you add a comment about how the user should resolve this (sync the file, then the parent directory).

Reply:
I refrained from doing so cause I’m not competent to give a portable answer.
On Linux, ``sync_all()`` on the file, then the directory is possible using just *std* (and
maybe *libc* for ``O_DIRECTORY``). Though how that maps to other OS which
tempfile supports, I have no idea.

Reply:
SGTM.